packages/keyring-api/src/api/transaction.ts

```diff
@@ -142,6 +142,14 @@ export enum TransactionType {
    * A swap transaction must be originated by the account.
    */
   Swap = 'swap',
+
+  /**
+   * The transaction is a bridge. It moves assets between chains.
+   *
+   * A bridge transaction must be originated by the account and it's a
```

Comment:
It's not clear to me from this description if a transaction marked as `bridge` is a send or a receive transaction. Should we have two types? `bridge-send` and `bridge-receive`?